### PR TITLE
support negative numbers

### PR DESCRIPTION
**Pull request by alanclarke**

### CODE REVIEW

**Comment by alanclarke** [comment]
@KidkArolis @giuse88

File: index.js [hunk 1]
```diff
@@ -21,6 +21,11 @@ function fromBase(input, base) {
   return val;
 }
 
+function prefix(num) {
+  if (String(num).charAt(0) === '-') return '-'
```
Comment (KidkArolis) [inline index.js]: would num < 0 be faster? :-"

Comment (alanclarke) [inline index.js]: yes, but it would break as the input can be a string or an int

**Comment by KidkArolis** [comment]
lgtm

**Pull request merged**